setup.py

```diff
@@ -228,6 +228,14 @@ def run(self):
         print("Unsupported command for openshift-ansible")
 
 
+def extract_requirements(filename):
+    """
+    Extracts requirements from a pip formatted requirements file.
+    """
+    with open(filename, 'r') as requirements_file:
+        return [x.split()[0] for x in requirements_file.readlines()]
```

Comment:
We don't even use it for normal testing, since we are doing the following in tox:
- setting skipdist=true without setting usedevelop=true, so we don't do an install step
- we are explicitly installing requirements.txt, setup-requirements.txt and test-requirements.txt

I'm good with dropping this, but I did like the idea of supporting the setuptools workflow if someone tried it. It would be nice to leverage setuptools for managing the dependencies as well, but I was not able to get it to use the cached packages instead of building them.

@ashcrow thoughts?